http-clients/apache-client/src/main/java/software/amazon/awssdk/http/apache/ApacheHttpClient.java

```diff
@@ -86,7 +86,7 @@
  * powerful synchronous client that adds an extra dependency and additional startup latency in exchange for more functionality,
  * like support for HTTP proxies.
  *
- * <p>See software.amazon.awssdk.http.urlconnection.UrlConnectionHttpClient for an alternative implementation.</p>
+ * <p>See software.amazon.awssdk.http.urlconnection.UrlConnectionHttpClient for an alternative implementation.</p> test
```

Comment:
nit: not sure what the " test" is

Reply:
This was added to trigger the CodeBuild during my test. Will remove when the PR is close to be merged.